package/mainflux/mainflux.mk

```diff
@@ -0,0 +1,30 @@
+################################################################################
+#
+# MAINFLUX
+#
+################################################################################
+
+MAINFLUX_VERSION = master
+MAINFLUX_SITE = $(call github,mainflux,mainflux,$(MAINFLUX_VERSION))
+MAINFLUX_LICENSE = Apache-2.0
+MAINFLUX_LICENSE_FILES = LICENSE
+
+MAINFLUX_BUILD_TARGETS = cmd/users
```

Comment:
Is it ok to put more than one build target? Because when we add the thing service from mainflux, will we add another package?

Reply:
Yes, it's expected. I think despite the mainflux being structured in microservices it remains a mono repository project, so it'd be better to create a unique Buildroot package for it (avoiding duplicated code and cloning the same repository twice to build specific parts of it).